keps/sig-node/2837-pod-level-resource-spec/README.md

```diff
@@ -1506,32 +1471,155 @@ KEPs.  The first change doesn’t present any user visible change, and if
 implemented, will in a small way reduce the effort for both of those KEPs by
 providing a single place to update the pod resource calculation.
 
-#### [Scoped for Beta] VPA
-
-TBD. Do not review for the alpha stage.
-
-#### [Scoped for Beta] Cluster Autoscaler
+#### [Scoped for Beta] HPA
+For accurate scaling decisions, HPA needs to be able to correctly calculate the
+resources requested by a pod, regardless of whether those requests are defined
+at the pod or container level. Currently, HPA calculates pod requests by simply
+aggregating the requests of all containers within a pod. To address this, HPA
+should leverage the helper method found at
+https://github.com/kubernetes/kubernetes/blob/988cf21f0975cf95444a619481c13d2503d8ec6a/staging/src/k8s.io/component-helpers/resource/helpers.go
+for more precise pod request computations. The changes are being worked on by
+sig-autoscaling: (#132237)[https://github.com/kubernetes/kubernetes/issues/132237]
+
+#### Cluster Autoscaler
```

Comment:
I could be wrong, but it looks like Karpenter, another node autoscaler project, has implemented its own logic to calculate pod resource requests. It might be worth suggesting to the Karpenter team that they consider using [k8s.io/component-helpers](https://pkg.go.dev/k8s.io/component-helpers/resource) or updating their implementation.
ref: https://github.com/kubernetes-sigs/karpenter/blob/v1.5.0/pkg/utils/resources/resources.go#L111-L144

/cc @jonathan-innis

Reply:
Added as a TODO